assets/src/styles/blocks/PageHeader.scss

```diff
@@ -0,0 +1,107 @@
+.is-pattern-p4_page-header {
+  margin-inline-start: calc(((100vw - 100%) / 2) * -1);
+  padding: 0;
+  width: 100vw;
+
+  img.wp-block-cover__image-background {
+    height: inherit;
+    min-width: 360px;
+    max-height: 270px;
+  }
+
+  .wp-block-group {
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+
+  .wp-block-group:first-child {
+    display: block;
+    margin-top: 233px;
+    max-width: 520px;
+  }
+
+  h1.has-background,
+  h2.has-background {
+    display: inline;
+    font-size: 2.5rem;
+    font-weight: 700;
+    letter-spacing: -1px;
```

Comment:
You could actually use a block setting for this:
<img width="295" alt="Screenshot 2022-06-22 at 10 42 34" src="https://user-images.githubusercontent.com/6949075/174984960-cd30c3d7-7323-4222-beea-2edcc12c43d1.png">

Reply:
But now that I think about it, wouldn't this affect accessibility? 🤔